spec/index.html

```diff
@@ -1149,12 +1615,300 @@ <h3>Overview</h3>
+            <p id="fig-nd-shared-hashes-alt">
+              The image represents the graph described in
+              <a href="#ex-nd-shared-hashes-input">
+                the following code block
+              </a>.</p>
+          </object>
+          <figcaption>An illustration of a graph resulting in shared hashes.<br/>
+            Image available in
+            <a href="shared-hashes.svg" title="SVG image of a graph resulting in shared hashes">
+              <abbr title="Scalable Vector Graphics">SVG</abbr>
+            </a>.</figcaption>
+        </figure>
+          
+        <pre id="ex-nd-shared-hashes-input"
+             data-transform="updateExample"
+             title="Graph with blank nodes resulting in shared hashes">
+          <!--
+            :p :q ****_:e0**** .
+            :p :q ****_:e1**** .
+            ****_:e0**** :p _:e2 .
+            ****_:e1**** :p _:e3 .
+            _:e2 :r _:e3 .
+          -->
+        </pre>
+
+        <p>The algorithm will be called twice, with each blank node
+           which did not result in a unique hash
+          from the <a href="#hash-1d-quads-algorithm">Hash First Degree Quads Algorithm</a>
+          (`_:e0` and `_:e1`).
+          The <a data-cite="INFRA#map-entry">map entry</a> for `_:e0` in the <a>blank node to quads map</a> results in the
+          following quads:</p>
+
+        <pre id="ex-nd-shared-hashes-e0-quads" data-transform="updateExample">
+          <!--
+            :p :q _:e0 .
+            _:e0 :p _:e2 .
+          -->
+        </pre>
+
+        <p>When called, the <var>temporary issuer</var> has the following mappings:</p>
+
+        <table id="ex-nd-shared-table-input-temporary-mapping-e0"
+               class="hash-related numbered">
+          <caption>Initial mappings in <var>temporary issuer</var> for `_:e0`</caption>
+          <thead><tr>
+            <th><var>original</var></th>
+            <th><var>canonical</var></th>
+          </tr></thead>
+          <tbody>
+            <tr><td>`_:e0`</td><td>`_:b0`</td></tr>
+          </tbody>
+        </table>
+
+        <p><a href="#hndq-3">Step 3</a> iterates over each of these quads
+          to find <a>blank node</a> <var>components</var>
+          to populate <var>H<sub>n</sub></var>
+          with the result of running the
+          <a href="#hash-related-blank-node">Hash Related Blank Node algorithm</a>
+          using the <var>position</var> of that <var>component</var> within the quad.</p>
+
+        <table id="ex-nd-shared-table-related-hash-e0"
+               class="hash-related numbered">
+          <caption>Hash for `_:e2`  related to `_:e0`</caption>
+          <thead><tr>
+            <th><var>related</var></th>
+            <th><var>quad</var></th>
+            <th><var>position</var></th>
+            <th><var>hash</var></th>
+          </tr></thead>
+          <tbody>
+            <tr>
+              <td>`_:e2`</td>
+              <td>`_:e0 :p _:e2 .`</td>
```

Comment:
Sadly, the tables are still hard to read clearly, even with the `th` centering ... because the wrapping of the `td` data has also been changed thereby. Either the quads in these tables need to use `&nbsp;` instead of ` ` characters, or the hashes need to be set to `abbr` (or to wrap without a wrapping character).

Reply:
I updated this particular table to use `&nbsp;` in the triple and `abbr` for the hash value, which makes it much more legible. Elsewhere, where length is not as much of an issue, I left the hash values intact.